lib/github_core.ml

```diff
@@ -355,6 +355,21 @@ module Make(Env : Github_s.Env)(Time : Github_s.Time)(CL : Cohttp_lwt.Client)
 
     let user_orgs ~user =
       Uri.of_string (Printf.sprintf "%s/users/%s/orgs" api user)
+
+    let repo_labels ~user ~repo =
+      Uri.of_string (Printf.sprintf "%s/repos/%s/%s/labels" api user repo)
+
+    let repo_label ~user ~repo ~name =
+      Uri.of_string (Printf.sprintf "%s/repos/%s/%s/labels/%s" api user repo name)
+
+    let issue_labels ~user ~repo ~num =
+      Uri.of_string (Printf.sprintf "%s/repos/%s/%s/issues/%d/labels" api user repo num)
+
+    let issue_label ~user ~repo ~num ~name =
+      Uri.of_string (Printf.sprintf "%s/repos/%s/%s/issues/%d/labels/%s" api user repo num name)
+
+    let milestone_labels ~user ~repo ~num =
+      Uri.of_string (Printf.sprintf "%s/repos/%s/%s/milestones/%d/labels" api user repo num)
```

Comment:
It would be nice to put these next to the high-level object endpoints they refer to. That is `repo_*` endpoints next to other `repo_*` endpoints, `issue_*` endpoints next to `issue_*` endpoints, etc.